qc-runner/src/classifier.r

```diff
@@ -100,17 +99,20 @@ task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsamp
       print("Info: empty-drops table of FDR threshold categories (# UMIs for a given threshold interval")
       print(table(obj_metadata$orig.ident, cut(obj_metadata$emptyDrops_FDR, breaks = c(-Inf,0,0.0001,0.01,0.1,0.5,1,Inf)), useNA="ifany"))
       print("How many barcodes should be filtered out for this sample (#FALSE):")
-      print(table(sample_subset$emptyDrops_FDR <= FDR))
+      
+      # prevents filtering of NA FDRs if FDR=1
+      ed_fdr <- sample_subset$emptyDrops_FDR
+      ed_fdr[is.na(ed_fdr)] <- 1
+      print(table(ed_fdr <= FDR))
```

Comment:
```suggestion
      print(paste0("Number of droplets with an emptyDrops-FDR <= ", FDR, " :"))
      print(table(ed_fdr <= FDR, useNA="ifany"))
```

Reply:
This info is already printed above a few lines. I'll bring it down so that it is more obvious